Game/Scripts/PlayerMoveScript.cpp

```diff
@@ -205,15 +227,40 @@ void PlayerMoveScript::Move(float deltaTime)
 	}
 }
 
-void PlayerMoveScript::MoveRotate(const float3& targetDirection, float deltaTime)
+void PlayerMoveScript::MoveRotate(float deltaTime)
 {
 	if (isDashing)
 	{
 		return;
 	}
 
+	//Look at enemy selected while attacking
+	AttackType currentAttack = bixAttackScript->GetCurrentAttackType();
+	GameObject* enemyGO = bixAttackScript->GetEnemyDetection();
+	if (enemyGO != nullptr && currentAttack != AttackType::NONE)
+	{
+		ComponentTransform* enemy = enemyGO->GetComponent<ComponentTransform>();
+		float3 vecForward = componentTransform->GetGlobalForward().Normalized();
+		float3 vecTowardsEnemy = (enemy->GetGlobalPosition() - componentTransform->GetGlobalPosition()).Normalized();
+		switch (currentAttack)
+		{
+		case AttackType::LIGHTNORMAL:
+			desiredRotation = vecForward + vecTowardsEnemy;
+			break;
+		case AttackType::HEAVYNORMAL:
+			desiredRotation = vecForward + vecTowardsEnemy;
+			break;
+		case AttackType::LIGHTFINISHER:
+			desiredRotation = vecForward + vecTowardsEnemy;
+			break;
```

Comment:
If these three are indeed intended to be the same, we could combine the cases.
```suggestion
		case AttackType::LIGHTNORMAL:
		case AttackType::HEAVYNORMAL:
		case AttackType::LIGHTFINISHER:
			desiredRotation = vecForward + vecTowardsEnemy;
			break;
```
Also, I don't know the `AttackType` enum off the top of my head, but do we need a `default` case here?

Reply:
Thanks Gerard! I changed bot things :)